app/controllers/v0/notifications/dismissed_statuses_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module V0
+  module Notifications
+    class DismissedStatusesController < ApplicationController
+      include Accountable
+
+      before_action :set_account, :set_notification
+
+      def show
+        if @notification
+          render json: @notification, serializer: DismissedStatusSerializer
+        else
+          raise Common::Exceptions::RecordNotFound.new(subject), 'No matching record found for that user'
+        end
+      end
+
+      private
+
+      def set_account
+        @account = create_user_account
+      end
+
+      def set_notification
+        @notification = Notification.find_by(account_id: @account.id, subject: subject)
+      end
+
+      def dismissed_statuses_params
+        params.permit(:subject, :dismissed_status, :status_effective_at)
```

Comment:
Are `:dismissed_status` and `:status_effective_at` being used?

Reply:
Those are staged for use in this [this ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17857) that is on deck.